### ISSUE TITLE
It feels like paper-autogrow-textarea should be in core-elements

### ISSUE THREAD

**Issue by davenotik** [Polymer/paper-input#140]
It seems paper-elements are elements that have some distinctive visual characteristics that relate to Material Design – things like `paper-input` and `paper-progress` look and feel like it's of Material Design. For something like an autogrowing textarea a la `paper-autogrow-textarea`, however, it seems more standard, or "core". There's nothing about it that is specific to Material Design. As I am using little of the distinctive Material Design but wanted a autogrowing textarea, I find myself using all core-elements except in this case where I'm using `paper-autogrow-textarea`. No biggie, just sharing FWIW.

**Comment by morethanreal** [comment]
I agree the `paper-autogrow-textarea` probably should be split from `paper-input` since it's more general purpose. We're going to reorganize the element set in the next major release, stay tuned!
